src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -33,9 +37,11 @@ internal class CurlHandler : HttpMessageHandler
 
         internal CurlHandler()
         {
+            Debug.Assert(s_CurlInitialized);
```

Comment:
I will be changing to static ctor but I didn't understand why the assert won't work. I am running my tests with this assertion.